client/post-editor/editor-ground-control/index.jsx

```diff
@@ -74,7 +78,9 @@ module.exports = React.createClass( {
 	},
 
 	setPostDate: function( date ) {
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
 		postActions.edit( { date: date ? date.format() : null } );
+		this.props.setDate( date );
```

Comment:
I just want to make sure this was a conscious decision to move this logic (setting to date.format() or null) into the setDate reducer. Maybe we should add a note into the placeholder reducer about this?